rocketpy/simulation/monte_carlo.py

```diff
@@ -654,9 +682,15 @@ def set_processed_results(self):
         """
         self.processed_results = {}
         for result, values in self.results.items():
-            mean = np.mean(values)
-            stdev = np.std(values)
-            self.processed_results[result] = (mean, stdev)
+            try:
+                if isinstance(values[0], float):
+                    mean = np.mean(values)
+                    stdev = np.std(values)
+                    self.processed_results[result] = (mean, stdev)
+                else:
+                    self.processed_results[result] = (None, None)
+            except TypeError:
+                self.processed_results[result] = (None, None)
```

Comment:
Now I understand a bit better the discussion on "Is the average even meaningful?". I would simply compute the average if the value "allows" it. Like:

```py
try:
    self.processed_results[result] = (np.mean(values), np.std(values))
except TypeError:
    self.processed_results[result] = (None, None)
```

Checking for a `float` seems odd to me, since `int`, `Decimal` etc could absolutely be desired for averaging things.

Reply:
I agree. Overall, it is not our responsibility to ensure every statistic output makes sense or not. Users should be responsible to critique what they want to average or not, regardless of the type of the variable.